entities/book.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+# books to read
+class Book
+  attr_reader :title, :author
+
+  def initialize
```

Comment:
please don't use faker in entities. initialize should accept title and author arguments

Reply:
also, you should add `validate` methods to your entities.
your initialize should look like:
```
validate title, author
@title = title
@author = author
```

Reply:
done